media-video/ffmpeg/ffmpeg-4.0.1.recipe

```diff
@@ -0,0 +1,239 @@
+SUMMARY="Audio and video recording, conversion, and streaming library"
+DESCRIPTION="FFmpeg is a complete, cross-platform solution to record, convert \
+and stream audio and video. It includes libavcodec - the leading audio/video \
+codec library."
+HOMEPAGE="https://ffmpeg.org/"
+COPYRIGHT="2000-2003 Fabrice Bellard
+	2003-2018 the FFmpeg developers"
+LICENSE="GNU LGPL v2.1
+	GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://ffmpeg.org/releases/ffmpeg-$portVersion.tar.xz"
+CHECKSUM_SHA256="605f5c01c60db35d3b617a79cabb2c7032412be243554602eeed1b628125c0ee"
+PATCHES="ffmpeg-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+ffmpegLibs="\
+	libavcodec \
+	libavdevice \
+	libavfilter \
+	libavformat \
+	libavresample \
+	libavutil \
+	libpostproc \
+	libswresample \
+	libswscale \
+	"
+# Do not sort alphabetically
+libavutilVersion="56.14.100"
+libavcodecVersion="58.18.100"
+libavformatVersion="58.12.100"
+libavdeviceVersion="58.3.100"
+libavfilterVersion="7.16.100"
+libavresampleVersion="4.0.0"
+libswscaleVersion="5.1.100"
+libswresampleVersion="3.1.100"
+libpostprocVersion="55.1.100"
+portVersionCompat="$portVersion compat >= 4"
+for i in $ffmpegLibs; do
+	eval "${i}VersionCompat=\"\$${i}Version compat >= \${${i}Version%%.*}\""
+done
+
+PROVIDES="
+	ffmpeg4$secondaryArchSuffix = $portVersionCompat
```

Comment:
You should **_either_** rename the recipe to `ffmpeg4-4.0.1.recipe` **_or_** `sed s/ffmpeg4/ffmpeg/` `PROVIDES{,_avdevice,_devel,_tools}`.

BTW, I'd recommend removing `ffmpeg-4.0.recipe` (and `patches/ffmpeg-4.0.patchset`.)